flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -225,7 +227,8 @@ int inferParallelism(FlinkInputFormat format, ScanContext context) {
       if (readableConfig.get(FlinkConfigOptions.TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM)) {
         int maxInferParallelism = readableConfig.get(FlinkConfigOptions
             .TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISM_MAX);
-        Preconditions.checkState(maxInferParallelism >= 1,
+        Preconditions.checkState(
+            maxInferParallelism >= 1,
```

Comment:
Nit: It seems like this change isn't updating anything.

Can we ensure we don't update a line / cause a diff where there's no change in the source code? This greatly helps people who maintain forks and have to `git cherry-pick` most commits.

Reply:
thx, will revert.